Re-enable visual tests in Chrome

limonte opened the pull request:
1. There's still some weird issue for Chrome visual tests in SauceLabs. But after separating chrome test to different task, it works fine.

<img src="https://i.imgur.com/pWZUYrL.png" width="500">

2. Remove `Vaadin.ThemableMixin`, it's already present in `vaadin-overlay`

<!-- Reviewable:start -->
---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/vaadin/vaadin-dialog/2)
<!-- Reviewable:end -->

tomivirkki commented:
Needs a rebase

---

Reviewed 6 of 6 files at r1.
Review status: all files reviewed at latest revision, 1 unresolved discussion.

---

*[vaadin-dialog.html, line 10 at r1](https://reviewable.io:443/reviews/vaadin/vaadin-dialog/2#-Krk5NoLFhjdAMh8YiWd:-Krk5NoLFhjdAMh8YiWe:b-x4wy4k) ([raw file](https://github.com/vaadin/vaadin-dialog/blob/13a6003258123d585c3622f26b988e896e279603/vaadin-dialog.html#L10)):*
> ```HTML
> <link rel="import" href="../polymer/lib/mixins/gesture-event-listeners.html">
> <link rel="import" href="../vaadin-overlay/vaadin-overlay.html">
> <link rel="import" href="../vaadin-themable-mixin/vaadin-themable-mixin.html">
> ```

Should be removed also

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/vaadin/vaadin-dialog/2#-:-Krk5RDFnWljXB7l_Pza:b-7blfmd)*
<!-- Sent from Reviewable.io -->

limonte commented:
Rebased.

---

Review status: 4 of 5 files reviewed at latest revision, 1 unresolved discussion.

---

*[vaadin-dialog.html, line 10 at r1](https://reviewable.io:443/reviews/vaadin/vaadin-dialog/2#-Krk5NoLFhjdAMh8YiWd:-KrkDpIJ4QTSGRXRMwmf:bss4a2b) ([raw file](https://github.com/vaadin/vaadin-dialog/blob/13a6003258123d585c3622f26b988e896e279603/vaadin-dialog.html#L10)):*
<details><summary><i>Previously, tomivirkki (Tomi Virkki) wrote…</i></summary><blockquote>

Should be removed also
</blockquote></details>

Removed.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/vaadin/vaadin-dialog/2#-:-KrkDo9Rmu9FQ3E0_lrU:bncvegj)*
<!-- Sent from Reviewable.io -->

YuriyVaadin commented:
<img class="emoji" title=":lgtm:" alt=":lgtm:" align="absmiddle" src="https://reviewable.io/lgtm.png" height="20" width="61"/>

---

Reviewed 3 of 6 files at r1, 1 of 2 files at r2.
Review status: all files reviewed at latest revision, 1 unresolved discussion.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/vaadin/vaadin-dialog/2#-:-KrkE-hKor4wk9dVSNbM:bnfp4nl)*
<!-- Sent from Reviewable.io -->

tomivirkki commented:
<img class="emoji" title=":lgtm:" alt=":lgtm:" align="absmiddle" src="https://reviewable.io/lgtm.png" height="20" width="61"/>

---

Reviewed 1 of 2 files at r2.
Review status: all files reviewed at latest revision, all discussions resolved, some commit checks pending.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/vaadin/vaadin-dialog/2#-:-KrkF76hdg_JwNJ6jId9:bnfp4nl)*
<!-- Sent from Reviewable.io -->
